recipe/meta.yaml

```diff
@@ -55,7 +55,8 @@ test:
   requires:
     - pip
     # to satisfy `transformers-cli --help` because it raises a warning of missing pytorch/tensorflow/flax for models.
-    - pytorch
+    - pytorch >=2
+    - blas * openblas     # [osx and x86_64]
```

Comment:
I think `blas * openblas     # [osx and x86_64]` should be moved to `run_constrained`.

Reply:
Good point, addressed